wdn/templates_4.0/scripts/extlatin.js

```diff
@@ -0,0 +1,14 @@
+define(['jquery', 'wdn', 'require'], function($, WDN, require) {
+
+    // Load Mercury ScreenSmart Extended Latin fonts from Cloud.typography
```

Comment:
I'd wrap all of this in the following to make it compatible with the `WDN.initializePlugin` signature:

```
var initd = false;

return {
    initializePlugin: function() {
        // protect against multiple initializations
        if (initd) {
            return;
        }
        initd = true;

        [...]
    }
}
```